cmd/account/cli.go

```diff
@@ -71,20 +71,29 @@ func (o *cliOptions) complete(cmd *cobra.Command) error {
 }
 
 func (o *cliOptions) run() error {
-	awsClient, err := o.k8sclusterresourcefactory.GetCloudProvider(o.verbose)
+	_, err := o.k8sclusterresourcefactory.GetCloudProvider(o.verbose)
```

Comment:
Is this call still needed if we aren't using its output?

Reply:
Yes. To do the k8s calls to get the target account information and return the creds required.